ocrd/ocrd/workspace_bagger.py

```diff
@@ -77,15 +82,40 @@ def _bag_mets_files(self, workspace, bagdir, ocrd_manifestation_depth, ocrd_mets
                 file_grp_dir = join(bagdir, 'data', f.fileGrp)
                 if not isdir(file_grp_dir):
                     makedirs(file_grp_dir)
-                self.resolver.download_to_directory(file_grp_dir, f.url, basename="%s%s" % (f.ID, f.extension))
-                f.url = join(f.fileGrp, f.ID + f.extension)
+
+                _basename = "%s%s" % (f.ID, f.extension)
+                _relpath = join(f.fileGrp, _basename)
+                self.resolver.download_to_directory(file_grp_dir, f.url, basename=_basename)
+                changed_urls[f.url] = _relpath
+                f.url = _relpath
 
             # save mets.xml
             with open(join(bagdir, 'data', ocrd_mets), 'wb') as f:
                 f.write(workspace.mets.to_xml())
 
+        # Walk through bagged workspace and fix the PAGE
+        # Page/@imageFilename and
+        # AlternativeImage/@filename
+        bag_workspace = Workspace(self.resolver, directory=join(bagdir, 'data'))
+        with pushd_popd(bag_workspace.directory):
+            for page_file in bag_workspace.mets.find_files(mimetype=MIMETYPE_PAGE):
+                pcgts = page_from_file(page_file)
+                changed = False
+                #  page_doc.set(imageFileName
+            #  for old, new in changed_urls:
+                for old, new in changed_urls.items():
+                    if pcgts.get_Page().imageFilename == old:
+                        pcgts.get_Page().imageFilename = new
+                        changed = True
+                    # TODO replace AlternativeImage, recursively...
```

Comment:
TODO indeed. @tboenig @bertsky Do you have some good sample PAGE with non-trivial AlternativeImage we could add to assets so I can test this and the solution to the newest iteration of #176 ?